tedana/combine.py

```diff
@@ -215,17 +221,19 @@ def make_optcom(data, tes, adaptive_mask, t2s=None, combmode='t2s', verbose=True
     data = data[mask, :, :]  # mask out unstable voxels/samples
     tes = np.array(tes)[np.newaxis, ...]  # (1 x E) array_like
     combined = np.zeros((data.shape[0], data.shape[2]))
+    report = True
     for echo in np.unique(adaptive_mask[mask]):
         echo_idx = adaptive_mask[mask] == echo
 
         if combmode == 'paid':
             combined[echo_idx, :] = _combine_paid(data[echo_idx, :echo, :],
-                                                  tes[:echo])
+                                                  tes[:echo], report=report)
         else:
             t2s_ = t2s[mask, ..., np.newaxis]  # mask out empty voxels/samples
 
             combined[echo_idx, :] = _combine_t2s(
-                data[echo_idx, :echo, :], tes[:, :echo], t2s_[echo_idx, ...])
+                data[echo_idx, :echo, :], tes[:, :echo], t2s_[echo_idx, ...], report=report)
+        report = False
```

Comment:
Should this actually be indented here? This means report is only turned on for the first iteration. Maybe that's the desired behavior, and I'm just confused, but I had thought you would want it on all iterations.

Reply:
We only want to add a line about how the echoes were combined to the report once, but if we leave it on for all iterations then we get duplicates of that line.

Reply:
Ah, yes, okay, I was just being confused. My mistake.